pkg/controllers/vdb/upgrade.go

```diff
@@ -680,33 +680,81 @@ func (i *UpgradeManager) closeAllSessions(ctx context.Context, pfacts *PodFacts)
 	return nil
 }
 
+const userHasDbadminRole = "(all_roles like 'dbadmin' " +
```

Comment:
If dbadmin is assigned to a role and this role is assinged to a user, we cannot see dbadmin in the user's all_roles colume. Can we use "has_role('dbadmin')"?

Reply:
we could... if we want to gather users and then run a sql statement for each user. `has_role` is a metafunction that can only go in a select statement on it's own. you can't even do `select has_role(user_name, 'pseudosuperuser') from users`. pseudosuperuser is a very powerful role. i can't imagine very many users would assign it to a role rather than just to a user directly. only a handful of users should ever have access to it.

Reply:
What if dbadmin role is assigned to a user?

Reply:
the pause only skips checking for pseudosuperuser, it was my mistake to check dbadmin.